src/plugins/data/public/query/saved_query/saved_query_service.ts

```diff
@@ -41,7 +43,9 @@ type SerializedSavedQueryAttributes = SavedObjectAttributes &
   };
 
 export const createSavedQueryService = (
```

Comment:
we should add a back log item to extend this service. i think at one point we were going to have a toggle to save the query with or without the dataset. which id imagine that is something we would want since in the past the saved query wouldn't replace the selected dataset (index pattern) it would just replace the current query